packages/SVG-Morphic.package/SVGStyleComposer.class/README.md

```diff
@@ -0,0 +1,11 @@
+A SVGStyleComposer is responsible to compose the initial style for a SVGComponent.
+
+Instance Variables
+	balloonStyle:		<Object>
+	svgStyle:		<Object>
+
+balloonStyle
+	- xxxxx
+
+svgStyle
+	- xxxxx
```

Comment:
Could you either provide more information on the instance variables or remove the auto-generated text